test/linter.js

```diff
@@ -82,13 +82,16 @@ async function getTagsFromConfig(config) {
 function execLinterCommand(args) {
   var cmd = `npx autorest@2.0.4152 --validation --azure-validator --message-format=json ${args}`.trim();
   console.log(`Executing: ${cmd}`);
+  var errorsFound = false;
   try {
     let result = execSync(cmd, { encoding: 'utf8', maxBuffer: 1024 * 1024 * 64 });
     console.error(result);
   } catch (err) {
-    throw new Error('Linter validation contains error(s)');
+    errorsFound = true;
```

Comment:
@sarangan12 I know we talked a bit offline, but could you provide details on the PR on the bug you're trying to fix? - like what issue you found and how it's not working with the current code. 

@dsgouda I believe we worked on adding looping through tags a while back and can't remember why we decided to exit early, do you?

Reply:
Created an issue and provided a detailed explanation and also linked the issue with this PR